.readthedocs.yaml

```diff
@@ -1,37 +1,13 @@
-# Read the Docs configuration file for Sphinx projects
-# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
-
-# Required
 version: 2
-
-# Set the OS, Python version and other tools you might need
 build:
-  os: ubuntu-lts-latest
+  os: ubuntu-22.04
```

Comment:
? I intentionally used `ubuntu-lts-latest` to minimize maintainer effort later... the odds of our builds breaking are low, and the consequences of it breaking are small since it's quickly caught via the RTD CI check and then easy enough for us to temp pin it at that point... but my guess is it'll never break even as ubuntu releases get bumped by RTD.

Not a big deal, just mentioning it... for low-volume repos, I'm more of a fan of floating...

Reply:
For now I'm just standardizing across projects, and pinning seemed better than not. What's nice is that once we're all standardized, we can use a tool like https://github.com/asottile/all-repos to update everything at once.

Reply:
Makes sense. I tend to forget about the broader pallets-eco...